.gitignore

```diff
@@ -22,7 +22,6 @@ pip-delete-this-directory.txt
 # Django stuff
 DatabaseManager.log*
 *.log
-local_settings.py
```

Comment:
Using purely environment variables now. DEBUG == False means it's inside docker (production)